### PR TITLE
Added `source` field to `subscriptions` table.

### PR DESCRIPTION
**Pull request by gbirke**
This is for https://github.com/wmde/fundraising/issues/1110

### CODE REVIEW

File: src/Entities/Subscription.php [hunk 1]
```diff
@@ -283,4 +290,20 @@ public function setHexConfirmationCode( $confirmationCode ) {
 		$this->confirmationCode = hex2bin( $confirmationCode );
 	}
 
+	/**
+	 * @param string $campaignCode
+	 * @return Subscription
+	 */
+	public function setCampaignCode( $campaignCode ) {
```
Comment (JeroenDeDauw) [inline src/Entities/Subscription.php]: You can php 7!

File: README.md [hunk 2]
```diff
@@ -47,6 +47,9 @@ run style checks use `composer cs`.
 ### Version 2.2.0 (dev)
 
 * Changed the minimum PHP version to 7.0
+* Added `campaign_code` field to `subscriptions` table. The campaign code is to differentiate between "remind me later" 
+  and other usages of the subscription functionality. The campaign code can be different from the "campaign" in the
```
Comment (JeroenDeDauw) [inline README.md]: I don't have a good overview of what these different classes of usages are

Comment (gbirke) [inline README.md]: At the moment, it's just "Remind me later" and "other", but there might be more usages in the future so a boolean flag seems to be inadvisable.

**Comment by gbirke** [comment]
TODO: We decided to call the field `subscriptionSource` instead of `campaignCode`, so this has to be reworked.

**Comment by JeroenDeDauw** [comment]
Reworked, ready for merge from my side

File: README.md [hunk 3]
```diff
@@ -56,6 +56,8 @@ run style checks use `composer cs`.
 #### Other changes
    
 * Changed the minimum PHP version to 7.0
+* Added `source_code` field to `subscriptions` table. This field indicates what led to the description,
```
Comment (KaiNissen) [inline README.md]: source~~_code~~

File: README.md [hunk 4]
```diff
@@ -56,6 +56,8 @@ run style checks use `composer cs`.
 #### Other changes
    
 * Changed the minimum PHP version to 7.0
+* Added `source` field to `subscriptions` table. This field indicates what led to the description,
```
Comment (gbirke) [inline README.md]: you mean "subscription"?

Comment (gbirke) [inline README.md]: Sorry, should've done a PR myself. I meant "This field indicates what led to the **sub**scription ..."

**Pull request merged**